core/bindings.rs

```diff
@@ -460,6 +464,54 @@ fn eval_context(
   rv.set(to_v8(tc_scope, output).unwrap());
 }
 
+/// This binding should be used if there's a custom console implementation
+/// available. Using it will make sure that proper stack frames are displayed
+/// in the inspector console.
+///
+/// Each method on console object should be bound to this function, eg:
+/// ```ignore
+/// function wrapConsole(consoleFromDeno, consoleFromV8) {
+///   const callConsole = core.callConsole;
+///
+///   for (const key of Object.keys(consoleFromV8)) {
+///     if (consoleFromDeno.hasOwnProperty(key)) {
+///       consoleFromDeno[key] = callConsole.bind(
+///         consoleFromDeno,
+///         consoleFromV8[key],
+///         consoleFromDeno[key],
+///       );
+///     }
+///   }
+/// }
+/// ```
+///
+/// Inspired by:
+/// https://github.com/nodejs/node/blob/1317252dfe8824fd9cfee125d2aaa94004db2f3b/src/inspector_js_api.cc#L194-L222
+fn call_console(
```

Comment:
For the record I think this is incredibly ugly, and it is incredibly frustrating that you can not mark stack frames as internal if they come from JS code. I can't come up with a better solution though, so I guess it is here to stay. Maybe can you rename to `double_call`? The function is not specific to console in any way implementation wise.

Reply:
Implementation is not console specific, but assertions are. I'd prefer to keep the name.

Reply:
They just assert the two things being passed are functions, right? Not console specific I think. But if you want to keep the name, I don't really care that much.